state/model.go

```diff
@@ -306,9 +330,15 @@ func (st *State) NewModel(args ModelArgs) (_ *Model, _ *State, err error) {
 	if err != nil {
 		return nil, nil, errors.Trace(err)
 	}
-	assertCloudRegionOp, err := validateCloudRegion(controllerCloud, args.CloudRegion)
-	if err != nil {
-		return nil, nil, errors.Trace(err)
+
+	var prereqOps []txn.Op
+
+	if args.Type == ModelTypeIAAS {
+		assertCloudRegionOp, err := validateCloudRegion(controllerCloud, args.CloudRegion)
+		if err != nil {
+			return nil, nil, errors.Trace(err)
+		}
+		prereqOps = append(prereqOps, assertCloudRegionOp)
 	}
```

Comment:
Should we do something here if the type is CAAS? Maybe if it's CAAS we should be checking that args.CloudRegion should be ""?